plugins/techdocs-node/api-report.md

```diff
@@ -71,7 +75,6 @@ export class Generators implements GeneratorBuilder {
     config: Config,
     options: {
       logger: Logger;
-      containerRunner: ContainerRunner;
```

Comment:
What about making it optional instead of removing? I'm afraid of breaking the code for people that where customizing it before.

Reply:
so this was supposed to be used by other modules such as `techdocs-backend` so it should be fine.
Ideally we would like to avoid supporting containerRunner, as techdocs-node will instantiate the docker runner internally by default.

Reply:
Bring this back up as I think we indeed need to keep this a non-breaking change. The `ContainerRunner` interface will still remain deprecated in `backend-common`, so we can rely on it for this code as long as we keep supporting the old backend system in TechDocs. I do think we can make this optional and use the `DockerContainerRunner` as the default implementation internally though.